presto-main/src/main/java/com/facebook/presto/operator/PagesRTreeIndex.java

```diff
@@ -191,8 +186,7 @@ private boolean testReferencePoint(Envelope probeEnvelope, OGCGeometry buildGeom
         return x >= extent.getXMin() && x < extent.getXMax() && y >= extent.getYMin() && y < extent.getYMax();
     }
 
-    private double getRadius(int joinPosition)
-    {
+    private double getRadius(int joinPosition) {
```

Comment:
The code style for Presto is to put these on the next line, and there's a build rule which will fail if this is not the case.